src/msg/async/dpdk/DPDK.cc

```diff
@@ -193,23 +193,23 @@ int DPDKDevice::init_port_start()
   // We want to support all available offload features
   // TODO: below features are implemented in 17.05, should support new ones
   const uint64_t tx_offloads_wanted =
-      DEV_TX_OFFLOAD_VLAN_INSERT      |
-      DEV_TX_OFFLOAD_IPV4_CKSUM       |
-      DEV_TX_OFFLOAD_UDP_CKSUM        |
-      DEV_TX_OFFLOAD_TCP_CKSUM        |
-      DEV_TX_OFFLOAD_SCTP_CKSUM       |
-      DEV_TX_OFFLOAD_TCP_TSO          |
-      DEV_TX_OFFLOAD_UDP_TSO          |
-      DEV_TX_OFFLOAD_OUTER_IPV4_CKSUM |
-      DEV_TX_OFFLOAD_QINQ_INSERT      |
-      DEV_TX_OFFLOAD_VXLAN_TNL_TSO    |
-      DEV_TX_OFFLOAD_GRE_TNL_TSO      |
-      DEV_TX_OFFLOAD_IPIP_TNL_TSO     |
-      DEV_TX_OFFLOAD_GENEVE_TNL_TSO   |
-      DEV_TX_OFFLOAD_MACSEC_INSERT;
+    DEV_TX_OFFLOAD_VLAN_INSERT      |
```

Comment:
please squash this change into "msg/async/dpdk: Porting seastar's init_port_start to fix to resolve ...". 

better off avoiding "fixing the fix" introduced in the same PR.